schema/meter_provider.json

```diff
@@ -103,13 +126,18 @@
             "type": "object",
             "additionalProperties": false,
             "minProperties": 1,
-            "maxProperties": 1,
```

Comment:
Should I keep this, and just make it `"maxProperties": 2`

Reply:
That seems fine for now.

Reply:
As in: It is fine to change it to `"maxProperties": 2`, or the current state is fine?

Reply:
Adding `maxProperties: 2` seems fine even if it doesn't perfectly provide the constraint we want.

Reply:
added